pyccl/halos/profiles/einasto.py

```diff
@@ -31,26 +32,26 @@ class HaloProfileEinasto(HaloProfile):
     By default, this profile is truncated at :math:`r = R_\\Delta(M)`.
 
     Args:
-        c_M_relation (:obj:`Concentration`): concentration-mass
+        concentration (:obj:`Concentration`): concentration-mass
             relation to use with this profile.
         truncated (bool): set to `True` if the profile should be
             truncated at :math:`r = R_\\Delta` (i.e. zero at larger
             radii.
         alpha (float, 'cosmo'): Set the Einasto alpha parameter or set to
             'cosmo' to calculate the value from cosmology. Default: 'cosmo'
     """
-    __repr_attrs__ = __eq_attrs__ = ("truncated", "alpha",
-                                     "cM", "precision_fftlog",)
-    name = 'Einasto'
+    __repr_attrs__ = ("truncated", "alpha",
+                      "precision_fftlog", "concentration", "normprof",)
```

Comment:
where did the `__eq_attrs__` go?

Reply:
and why have the names disappeared?

Reply:
magic! I have no idea where eq_attrs went! And when I look at the commits one-by-one it looks like I never removed them, so I don't know how they decided to go.

Reply:
I have removed all names from the profiles, after introducing `CCLNamedClass`. Names weren't used by anything, no one will ever do `HaloProfile.from_name` so there is no point in defining useless class variables.